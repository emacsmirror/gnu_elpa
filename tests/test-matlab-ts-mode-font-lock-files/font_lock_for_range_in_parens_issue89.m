% -*- matlab-ts -*-

% see https://github.com/acristoffers/tree-sitter-matlab/issues/89

for (i = 1:5)
    disp(i)
end
