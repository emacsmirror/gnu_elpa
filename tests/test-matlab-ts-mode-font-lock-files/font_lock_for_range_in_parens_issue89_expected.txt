c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

kkk bV o nDnb
    BBBBbBb
kkk
